docs/interactions/structured-messages/index.md

```diff
@@ -39,7 +39,7 @@ Engage Digital provides an omnichannel API to create structured messages. Since
   | **Google Business Messages** | **<span style="color:green">Yes</span>** | **<span style="color:green">Yes</span>** | **<span style="color:green">Yes</span>** | **<span style="color:green">Yes</span>** | N/A | N/A |  N/A |
   | **Source SDK** | **<span style="color:green">Yes</span>** | **<span style="color:green">Yes</span>** | **<span style="color:green">Yes</span>** | **<span style="color:green">Yes</span>** | N/A | N/A |  N/A |
   | **Twitter Direct Messages** | **<span style="color:orange">Not yet</span>** | **<span style="color:orange">Not yet</span>** | **<span style="color:orange">Not yet</span>** | N/A | N/A | N/A |  N/A |
-  | **WhatsApp** | **<span style="color:orange">Not yet</span>** | **<span style="color:orange">Not yet</span>** | **<span style="color:orange">Not yet</span>** | N/A | N/A | N/A |  N/A |
+  | **WhatsApp** | **<span style="color:green">Yes</span>** | **<span style="color:orange">Not yet</span>** | **<span style="color:green">Yes</span>** | N/A | N/A | N/A |  N/A |
```

Comment:
I think you can also change the rich link support to N/A

Reply:
changed, I also sorted it alphabetically